tests/test_enterprise/management/test_create_enterprise_customer_admins.py

```diff
@@ -0,0 +1,80 @@
+"""
+Tests for the Django management command `create_enterprise_customer_admins`.
+"""
+
+import logging
+
+import ddt
+from pytest import mark
+from testfixtures import LogCapture
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from enterprise.models import (
+    EnterpriseCustomerAdmin,
+    SystemWideEnterpriseUserRoleAssignment,
+)
+from enterprise import roles_api
+from test_utils.factories import (
+    EnterpriseCustomerFactory,
+    EnterpriseCustomerUserFactory,
+    UserFactory,
+)
+
+
+@mark.django_db
+@ddt.ddt
+class CreateEnterpriseCustomerAdminsCommandTests(TestCase):
```

Comment:
I like the tests you wrote, clean and concise while catching what's important.

There could maybe be some sort of test that checks that if an error occurs on one record it's swallowed and logged instead of throwing an error and crashing the whole job, but I think it's optional - since it's mostly a one-time task. Up to you whether you want to implement that.

I don't see any worrisome edge cases otherwise, I think the tests are sufficient.